src/main/perl/LV.pm

```diff
@@ -381,19 +381,29 @@ sub print_ks
     $self->{volume_group}->print_ks;
     print "\n";
 
-    my $useexisting = '';
+    my @format = $self->ksfsformat($fs);
```

Comment:
if this i sthe same code block as below, move it in method in Blockdevices and call the method instead of copy/pasting

Reply:
I have some memories of trying that and then having to revert, but I don't remember why. I'll take another look at the code tomorrow.